src/services/refactors/convertExport.ts

```diff
@@ -31,7 +31,8 @@ namespace ts.refactor {
         const { file } = context;
         const span = getRefactorContextSpan(context);
         const token = getTokenAtPosition(file, span.start);
-        const exportNode = getParentNodeInSpan(token, file, span);
+        // If the span is entirely contained in an export node, check that node.

```

Comment:
This seems unrelated?

Reply:
This and the changes in `convertImport.ts` are just offer refactors more often. They should probably go in a separate PR.